moped-editor/src/queries/subprojects.js

```diff
@@ -2,14 +2,19 @@ import { gql } from "@apollo/client";
 
 export const SUBPROJECT_QUERY = gql`
   query SubprojectSummary($projectId: Int) {
-    subprojects: moped_project(where: { project_id: { _eq: $projectId } }) {
-      moped_projects(where: { is_deleted: { _eq: false } }) {
-        project_name
-        project_id
-        status_id
-        current_phase
```

Comment:
this query had an extra layer of nesting that didn't seem necessary, but i might be missing something. the feature seems to be working fine after i fixed up the data accessors.